cmd/node/node.go

```diff
@@ -383,19 +382,21 @@ func (app *SpacemeshApp) Start(cmd *cobra.Command, args []string) {
 
 	app.unregisterOracle = func() { oracleClient.Unregister(true, pub.String()) }
 
-	//bo := oracle.NewBlockOracleFromClient(oracleClient, int(app.Config.CONSENSUS.NodesPerLayer))
-	nodesPerLayer := app.Config.CONSENSUS.NodesPerLayer
-	layersPerEpoch := app.Config.CONSENSUS.LayersPerEpoch
-	activationDb := &activation.ActivationDb{}      // TODO: initialize properly
-	beaconProvider := &oracle.EpochBeaconProvider{} // TODO: initialize properly
-	vrfSigner := crypto.NewVRFSigner(nil)           // TODO: use VRF private key
-	nodeID := types.NodeId{Key: "x"}
-	bo := oracle.NewMinerBlockOracle(nodesPerLayer, layersPerEpoch, activationDb, beaconProvider, vrfSigner, nodeID)
+	nodeID := types.NodeId{Key: pub.String()}
+	bo := oracle.NewBlockOracleFromClient(oracleClient, int(app.Config.CONSENSUS.NodesPerLayer), nodeID)
+	//nodesPerLayer := app.Config.CONSENSUS.NodesPerLayer
+	//layersPerEpoch := app.Config.CONSENSUS.LayersPerEpoch
+	//activationDb := &activation.ActivationDb{Atxs: database.NewMemDatabase()} // TODO: initialize properly
+	//beaconProvider := &oracle.EpochBeaconProvider{}                           // TODO: initialize properly
+	//vrfSigner := crypto.NewVRFSigner(nil)                                     // TODO: use VRF private key
+	//nodeID := types.NodeId{Key: "x"}
+	//bo := oracle.NewMinerBlockOracle(nodesPerLayer, layersPerEpoch, activationDb, beaconProvider, vrfSigner, nodeID)
 	hareOracle := oracle.NewHareOracleFromClient(oracleClient)
 
 	apiConf := &app.Config.API
 
-	err = app.initServices("x", swarm, "/tmp/", sgn, bo, nil, hareOracle, 50)
+	validatorMock := sync.BlockValidatorMock{}
+	err = app.initServices(nodeID, swarm, "/tmp/", sgn, bo, validatorMock, hareOracle, 50)
```

Comment:
please change 50 to app.Config.LayerAvgSize
this must have gotten lost when we switched to cmds